### DISCUSSION TITLE
3DSlicer MONAI Label UI

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#1517] [Q&A]
Hi there!
Could you help with several questions about 3DSlicer MONAI Label UI below?
1. In the latest version, we see "Section" is added (see screenshot below), which didn't exist in previous versions. We're confused on how we shall choose from the modes within the "Section". E.g., if we choose "activelearning", can we still do training like in previous version? What does "scoring" mode mean?

![Options](https://github.com/Project-MONAI/MONAILabel/assets/90806807/d97dffef-fee9-4269-ac43-341f3c3a4e2c)

2. The "Options" panel allows users to change training hyperparameters easily, but there's no "optimizer" and "learning rate" knobs now (see screenshot below). Will those be added? And how can we tune those hyperparameters for now?
![lr](https://github.com/Project-MONAI/MONAILabel/assets/90806807/a6f6cff4-35e0-4751-85d0-9bc5fa7f9fa0)
![lr_2](https://github.com/Project-MONAI/MONAILabel/assets/90806807/63578104-8414-4c6c-a47f-6a97b3cb863e)

3.  The "Option" panel in MONAI Label is not changed when I changed the argument of `name`, `batch_size` and `max_epochs` in `/apps/deepedit/main.py`. Is this expected?

@SanghoonKang503 for visibility.

**Comment by tangy5** [comment]
Hi @hw-ju 
1. The sections refer to different actions for monailabel, you could choose an action and make config changes on the panel. if we choose Yes, if you choose "activelearning", you can go back and do training. The section and option panel is just a UI for change configurations of certain parameters. 
2. Currently, we didn't put optimzer related hyperparameter in the options, you could modify the args in the python or bundle files where you defined. For example, in the radiology app, the "segmentation" model, the optimizer is defined here: https://github.com/Project-MONAI/MONAILabel/blob/1d034726c655102cf612053842b1c45727966612/sample-apps/radiology/lib/trainers/segmentation.py#L61

3. Once you changed the variables in the options panel, these variables will pass to the request and send to training or infer tasks. It will not related to the "main.py" .

Hope these answers help!

**Reply by hw-ju** [reply]
@tangy5 Thank you so much for helping! Very informative!

For the 1st bullet, I think I get the idea "The section and option panel is just a UI for change configurations of certain parameters.", but could you explain more about 
```
if we choose Yes, if you choose "activelearning", you can go back and do training.
```
